### DISCUSSION TITLE
Keep gettting errors while creating collection offers via sdk

### DISCUSSION THREAD

**Discussion by wainesiro** [ProjectOpenSea/opensea-js#1505] [Q&A]
Hellp i have been using Sdk to make collection offers and i keep getting different errors and it automatically gets ok and bids on retry. the error looks like "Error: Server Error: You have attempted to create a order, but you provided an offer item or consideration item with a 0 start amount. Remove this 0-amount item if this was intended." or also "ITS NOT A VALID JSON"

**Comment by ryanio** [comment]
hey there, i can't really help you further unless you share the exact code you are running and errors you are receiving back, to help debug.

have you followed the quick start guide? https://github.com/ProjectOpenSea/opensea-js/blob/main/developerDocs/quick-start.md

**Reply by ryanio** [reply]
also if you use TypeScript you should get some more feedback if the values and types you are passing to the createCollectionOffer() function are valid
